src/test/java/teammates/test/cases/browsertests/InstructorFeedbackPageUiTest.java

```diff
@@ -279,7 +279,7 @@ public void testAddAction() throws Exception {
         newSession.setInstructions(new Text("<p>Please answer all the given questions.</p>"));
         newSession.setGracePeriod(15);
         
-        newSession.setFeedbackSessionType(FeedbackSessionType.PRIVATE);
+        newSession.setFeedbackSessionType(FeedbackSessionType.STANDARD);
```

Comment:
You should not need to change this or any HTML file for testing purpose. Refactoring should not modify the action's behaviour.